server/api/locationWeatherAPI.js

```diff
@@ -0,0 +1,32 @@
+const openWeatherKey = require("./apiKeys/openWeather");
+
+const convertZipCode = async (url) => {
+  try {
+    const result = await fetch(url).then((data) => data.json());
+    const data = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?lat=${result.lat}&lon=${result.lon}&appid=${openWeatherKey}&units=imperial`
+    ).then((data) => data.json());
+    console.log(`result: `, result);
+
+    const tempObj = {
+      currTemp: data.main.temp,
+      maxTemp: data.main.temp_max,
+      minTemp: data.main.temp_min,
+    };
+    return tempObj;
+  } catch (err) {
+    console.log("ERROR", err);
```

Comment:
I think this is good for now, but do we want to do something more meaningful with this error later?